### ISSUE TITLE
Coincidence Tier Message required fields

### ISSUE THREAD

**Issue by KaraMelih** [SNEWS2/SNEWS_Publishing_Tools#115]
https://github.com/SNEWS2/SNEWS_Publishing_Tools/blob/62a62526aa3a3dd1147dad721893af53df520e8a/snews_pt/messages.py#L268

It looks like the coincidence tier messages only require neutrino time, however, the validation checks for the p value no matter what. As a result;
```
messages.SNEWSMessageBuilder(neutrino_time=datetime.utcnow())
```
raises;
```
TypeError: '<=' not supported between instances of 'float' and 'NoneType'
```

which is not informative, and we wanted avoid exactly this when we implemented validation-upon-creation

**Comment by KaraMelih** [comment]
Addressed in #116

**Comment by habig** [comment]
Melih deploy a small "don't need p_value" fix independent of the heartbeat thread mentioned above.
